### DISCUSSION TITLE
Different DEs or a base image

### DISCUSSION THREAD

**Discussion by gr4phite** [Ultra980/danora-linux#6] [Ideas]
We could make different desktops or a base image for customization. I made https://github.com/Ultra980/danora-linux/commit/316f06356192e18bc228c193b8565e5b3ffd0e56 with this purpose.

**Comment by gr4phite** [comment]
My idea is to make a new build workflow that changes the `FEDORA_DISTRO` arg to `fedora-silverblue` or `fcos` (Fedora CoreOS).

**Comment by gr4phite** [comment]
The base/fcos workflow doesn't work, the current repo doesn't support version tags. I also asked on fedora discussions: https://discussion.fedoraproject.org/t/is-there-a-coreos-oci-image-repository-that-has-version-tags/45716

**Comment by gr4phite** [comment]
The Gnome version needs testing. @AndryCake see https://github.com/Ultra980/danora-linux/issues/7

**Reply by TheGreenGuy5** [reply]
Is Gnome version live? (Do we have any Gnome image?)

**Reply by gr4phite** [reply]
Yes.

**Reply by gr4phite** [reply]
https://github.com/Ultra980/danora-linux/commit/c77c7bd28d5d07e3e2878c593489c65acbb02cd8

**Comment by gr4phite** [comment]
See https://github.com/Ultra980/danora-linux/commit/c77c7bd28d5d07e3e2878c593489c65acbb02cd8

**Comment by gr4phite** [comment]
Welp, now the Gnome image is the only one that builds.
